symphony/chain.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "symphonytestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Symphony Testnet",
```

Comment:
Was this just copy and pasted and now reviewed? It says "testnet" and "Symphony Testnet" but being put into a directory like it's a mainnet